roles/libselinux/handlers/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: uninstall libselinux-python
+  yum:
+   name=libselinux-python
+   state=removed
```

Comment:
Please use new-style yaml syntax in your playbooks:

    - name: uninstall libselinux-python
      yum:
        name: libselinux-python
        state: removed